### ISSUE TITLE
.kdbx file support

### ISSUE THREAD

**Issue by Minibus93** [yogeshpaliyal/KeyPass#47]
**Is your feature request related to a problem? Please describe.**
I'd like to use KeyPass, but it doesn't support .kdbx files and I'm using KeePass on pc. Mainteining two password files is not ideal

**Describe the solution you'd like**
A .kdbx file support would fix the inconvenience

**Describe alternatives you've considered**
There aren't alternative. If i want to use KeyPass I should manually add more than 200 credentials and maintein them by adding credentials manually when I add a new entry on KeePass for pc

**Additional context**

**Comment by yogeshpaliyal** [comment]
Thanks for letting us know, will check if it is possible or not and let you know.

**Comment by IJR222** [comment]
I agree that this is important. The "Keypass" name makes it sound like a KeePass implementation.

**Comment by 4mohamedaliii** [comment]
Are you working on this?

**Comment by mylespm** [comment]
Any update on this?

**Comment by yogeshpaliyal** [comment]
Added Support for Importing CSV from KeePass

**Comment by IJR222** [comment]
I don't believe that is an option in keepassdx

**Comment by LovecraftianGodsKiller** [comment]
> How to generate a CSV file from KeepassDX?

If you have access to a computer, with or without admin, you might be able to with [KeePassXC](https://github.com/keepassxreboot/keepassxc/releases).
Download the normal exe if you have admin or the zip if you don't

**Comment by shuvashish76** [comment]
https://github.com/Kunzisoft/KeePassDX/wiki/Import-and-Export#hard-way
